libplatsupport/src/plat/imx6/clock.c

```diff
@@ -576,11 +637,16 @@ static struct clock mmdc_ch0_clk = { CLK_OPS(MMDC_CH0, mmdc_ch0, NULL) };
 
 static freq_t _ahb_get_freq(clk_t *clk)
 {
+    /* ToDo: read divider from CBCDR.AHB_PODF */
     return clk_get_freq(clk->parent) / 4;
 }
 
 static freq_t _ahb_set_freq(clk_t *clk, freq_t hz)
 {
+    /* ToDo: we assume the default value CBCDR.AHB_PODF = b011 has not been
```

Comment:
Same here: Is this TODO intentional and not something that was forgotten?

Reply:
This is intentional in this comment I've just refactored the code and added comments to clarify the assumption made here.